htdocs/install/page_modcheck.php

```diff
@@ -63,37 +64,51 @@ function xoDiagIfWritable( $path) {
 	return xoDiag( $error ? -1 : 1, $error ? 'Not writable' : 'Writable' );
 }
 
+function imCheckRequirements()
```

Comment:
nice solution!

Reply:
Thanks! I was trying to separate the logic from the presentation. I'm sure it can be improved in further iterations

Reply:
Here is how we were approaching it at the beginning of 2.0 - https://github.com/skenow/impresscms/tree/features/installer2.0/htdocs